src/commands/openInPortal.ts

```diff
@@ -13,5 +13,5 @@ export async function openInPortal(context: IActionContext, node?: AzExtTreeItem
         node = await ext.rgApi.tree.showTreeItemPicker<AzExtTreeItem>(new RegExp(ResolvedFunctionAppResource.productionContextValue), context);
     }
 
-    await uiOpenInPortal(node, node.fullId);
+    await uiOpenInPortal(node, `${nonNullProp(node, 'parent').parent?.id}/Deployments/${nonNullProp(node, 'id')}`);
```

Comment:
What's up with this? Doesn't this only work for very specific tree nodes?

Reply:
The Functions open in portal command is now only used for deployment tree items, we could rename the command to make that more clear.

Reply:
Could you add a comment to that effect?  This would be super confusing.  Or maybe just change `openInPortal` to `openDeploymentInPortal`